src/components/ToscaDataViewer/index.jsx

```diff
@@ -52,7 +54,7 @@ const ToscaDataViewer = (props) => {
         index={res._index}
         id={res._id}
       />
-      {browseUrl ? (
+      {(!(hide_browse_url) && browseUrl) ? (
```

Comment:
no need to wrap `hide_browse_url` in parenthesis:
```js
!hide_browse_url && browseUrl
```

Reply:
Done @DustinKLo